core/src/main/scala/libretto/scaletto/StarterKit.scala

```diff
@@ -35,7 +36,7 @@ abstract class AbstractStarterKit(
   val invertLib: InvertLib[coreLib.type] =
     InvertLib(coreLib)
 
-  def executor(blockingExecutor: JExecutor)(implicit
+  def executor(blockingExecutor: JExecutor)(using
```

Comment:
This API seems messed up, imho.

Why it's messed up should be clear from the usage sides. The look fishy.

IDK, but maybe this should be just a regular param? Than all the `using`s on call side wouldn't be needed. Also passing parts of the executor looks "funny".

The whole runtime needs anyway some love. I'll open issues…

Reply:
Let's use a regular explicit parameter.

I agree it needs some more love.